OrbitGl/ThreadTrack.cpp

```diff
@@ -53,7 +53,6 @@ std::string ThreadTrack::GetBoxTooltip(PickingId id) const {
 
   const FunctionInfo* func =
       GOrbitApp->GetCaptureData().GetSelectedFunction(text_box->GetTimerInfo().function_address());
-  CHECK(func != nullptr);
```

Comment:
Is there a way to keep the `CHECK`? Like `CHECK(function != nullptr || introspection)`?

Reply:
Done.